packages/opds-web-client/src/components/Root.tsx

```diff
@@ -113,6 +122,7 @@ export class Root extends React.Component<RootProps, RootState> {
     let showBreadcrumbs = this.props.collectionData && breadcrumbsLinks.length > 0;
     let showSearch = this.props.collectionData && this.props.collectionData.search;
     let showFooter = this.props.collectionData && Footer;
+    let showEntryPointsTabs = !!EntryPointsTabs;
```

Comment:
I think the entry points should only be displayed if you're showing a collection rather than a book.

Reply:
I guess you're doing something with the bookPage prop, maybe I'll have to wait for the next pr :)

Reply:
Yes, but now that I think about it, it makes more sense to just not render the component as a whole instead of rendering it but then checking if a book is being displayed to not render parts of it.